$OpenBSD: patch-src_chrome_content_popup_xul,v 1.3 2018/10/03 14:57:09 sthen Exp $

Get rid of "Check for Tor Browser Updates" button in UI since it can't
do anything useful right now.
Index: src/chrome/content/popup.xul
--- src/chrome/content/popup.xul.orig
+++ src/chrome/content/popup.xul
@@ -36,12 +36,6 @@
                   label="&torbutton.context_menu.networksettings;"
                   accesskey="&torbutton.context_menu.networksettings.key;"
                   oncommand="torbutton_open_network_settings()"/>
-        <menuseparator id="torbutton-checkForUpdateSeparator"/>
-        <menuitem id="torbutton-checkForUpdate"
-                  label="&torbutton.context_menu.downloadUpdate;"
-                  accesskey="&torbutton.context_menu.downloadUpdate.key;"
-                  insertafter="context-stop"
-                  oncommand="torbutton_check_for_update()"/>
       </vbox>
     </hbox>
   </panel>
